chainer/functions/normalization/batch_normalization.py

```diff
@@ -567,6 +635,14 @@ def batch_normalization(x, gamma, beta, **kwargs):
             be ``None``.
         decay (float): Decay rate of moving average. It is used during
             training.
+        axis (int or tuple of int): Axis over which normalization is
+            performed. When axis is None, it is determined from input
```

Comment:
``None`` (use double quotes).  The same is true of other programming expressions.